groovy/postInit/mod/SusyCore.groovy

```diff
@@ -103,3 +103,199 @@ mods.gregtech.assembler.recipeBuilder()
 mods.gregtech.forge_hammer.removeByInput(16, [item('gregtech:ore_sulfur_0')], null)
 // Crushed Sulfur Ore * 2
 mods.gregtech.macerator.removeByInput(2, [item('gregtech:ore_sulfur_0')], null)
+
+// Armors
+crafting.addShaped("susy:simple_gas_mask", item('susy:susy_armor_item').withNbt(['damage': 0.0D]), [
+    [item('gregtech:meta_item_1', 438), item('minecraft:paper'), item('gregtech:meta_item_1', 438)],
+    [ore('string'), item('gregtech:meta_dust', 271), ore('string')],
```

Comment:
what dust is this?

Reply:
I think it's carbon/charcoal dust, if I'm not mistaken?

Reply:
could work